### PR TITLE
Allow override of deployment replicas value

### PR DESCRIPTION
**Pull request by funkypenguin**
This PR allows the helm-chart deployer the ability to customize the amount of replicas in the deployment. A potential future enhancement in this area would be to add a PodDisruptionBudget, as well as affinity settings, to accommodate availability requirements on an HA cluster.

### CODE REVIEW

File: helm/fdb-operator/values.yaml [hunk 1]
```diff
@@ -4,3 +4,4 @@ operator:
   tag: 0.24.0
   role: fdb-kubernetes-operator-manager-role
   rolebinding: fdb-kubernetes-operator-manager-rolebinding
+  replicas: 1
```
Comment (johscheuer) [inline helm/fdb-operator/values.yaml:7]: NIT: missing newline

**Review by johscheuer — CHANGES_REQUESTED** [review]

**Comment by johscheuer** [comment]
We can add a `PodDisruptionBudget` optionally but the availability of the operator doesn't directly effect the availability of the database itself.

File: helm/fdb-operator/values.yaml (hunk 1 above)
Comment (funkypenguin) [reply helm/fdb-operator/values.yaml]: Fixed :)

**Comment by funkypenguin** [comment]
> We can add a `PodDisruptionBudget` optionally but the availability of the operator doesn't directly effect the availability of the database itself.

Agreed, it's a nice-to-have in future, and I felt that adding it to this changeset was unnecessary :)

**Review by johscheuer — APPROVED** [review]
👍

**Comment by johscheuer** [comment]
Once the conflict is solved I merge the PR. Thanks!

**Comment by funkypenguin** [comment]
Conflict resolved ;)

**Comment by johscheuer** [comment]
Thanks!

**Pull request merged**